sage-ios/SAGE/SAGE/SignUpRequestsViewController.swift

```diff
@@ -74,7 +79,14 @@ class SignUpRequestsViewController: UITableViewController {
             self.activityIndicator.stopAnimating()
             self.refreshControl?.endRefreshing()
             
+            if self.requests == nil || self.requests?.count == 0 {
+                self.showNoContentView()
+            } else {
+                self.hideNoContentView()
+            }
+            
             }) { (errorMessage) -> Void in
+                self.showNoContentView()
```

Comment:
hmm this should also have 

self.activityIndicator.stopAnimating()
self.refreshControl?.endRefreshing()